rir/src/compiler/translations/pir_2_rir.cpp

```diff
@@ -694,8 +699,12 @@ size_t Pir2Rir::compileCode(Context& ctx, Code* code) {
                             std::cerr << " (" << tagToStr(what->tag) << ")\n";
                             assert(false);
                         }
-                        if (!alloc.onStack(what))
+                        if (!alloc.onStack(what)) {
                             cs << BC::ldloc(alloc[what]);
+                            std::stringstream ss;
+                            what->printRef(ss);
+                            cs.addSrc(Rf_install(ss.str().c_str()));
```

Comment:
What is this? We add source to the load that is just the symbol of the PIR value? Cannot this blow up if someone accesses this from R?

Reply:
that's just the name of the pir value, it's very useful for debugging... If you want I can put it behind a debug flag